components/test_raftstore-v2/src/util.rs

```diff
@@ -139,6 +143,13 @@ pub fn configure_for_encryption(config: &mut Config) {
     cfg.master_key = test_util::new_test_file_master_key(manifest_dir);
 }
 
+pub fn configure_for_request_snapshot(config: &mut Config) {
+    // We don't want to generate snapshots due to compact log.
+    config.raft_store.raft_log_gc_threshold = 1000;
+    config.raft_store.raft_log_gc_count_limit = Some(1000);
+    config.raft_store.raft_log_gc_size_limit = Some(ReadableSize::mb(20));
+}
```

Comment:
No need duplicate.

Reply:
why it's duplicate?

Reply:
Same as `test_raftstore::configure_for_request_snapshot`.